src/core/compiler.jl

```diff
@@ -191,13 +192,21 @@ function model_generator(x = nothing, y = nothing)
     function inner_function(vi::Turing.VarInfo, sampler::Turing.AbstractSampler, model)
         local x
         if isdefined(model.data, :x)
```

Comment:
I probably miss something in previous PRs but why did we use `:x` and `:y` specifically here?

Reply:
It is an example with `x` and `y` as data variables defined with the `@model` macro.

Reply:
Oh yes sorry for my stupidness.